runtime/shared_common/shrinit.cpp

```diff
@@ -3177,6 +3177,44 @@ j9shr_isBCIEnabled(J9JavaVM *vm)
 	return (0 != (vm->sharedClassConfig->runtimeFlags & J9SHR_RUNTIMEFLAG_ENABLE_BCI));
 }
 
+static UDATA
```

Comment:
We always declare the static functions here: https://github.com/eclipse-openj9/openj9/blob/0985ff3f82bfdf1944c81f6242466fcbae2308b5/runtime/shared_common/shrinit.cpp#L394-L412